### PR TITLE
docs(router): Updated docs to reflect alpha 7 release

### PR DESCRIPTION
**Pull request by brandonroberts**
cc @filipesilva

### CODE REVIEW

**Comment by filipesilva** [comment]
lgtm as well.

**Comment by Foxandxss** [comment]
Can you fix all the lint issues? `gulp lint`.

Thanks.

**Comment by brandonroberts** [comment]
Sure. Thanks

**Comment by Foxandxss** [comment]
There is one issue on the tests too. Please fix.

File: public/docs/ts/latest/guide/router.jade [hunk 1]
```diff
@@ -401,7 +404,7 @@ h4#register-providers Register routing in bootstrap
   Providing the router providers at the root makes the Router available everywhere in our application.
```
Comment (Foxandxss) [inline public/docs/ts/latest/guide/router.jade]: Not precisely from this update. We normally call "root" to the AppComponent and not the bootstrap. Wouldn't be better to say "providers at boostrap makes the ..."

**Comment by brandonroberts** [comment]
Lint and testing issues fixed

**Comment by naomiblack** [comment]
merged and live

**Pull request closed without merging**